### PR TITLE
map recipes

### PR DESCRIPTION
**Pull request by eristhea**
refactors:
- switched up recipe book naming to better match moj
- fixed a few errors

### CODE REVIEW

**Comment by eristhea** [comment]
finally ready for review!

File: mappings/net/minecraft/recipe/RecipeMatcher.mapping [hunk 1]
```diff
@@ -1,34 +1,100 @@
...
 	METHOD m_okheizlq consume (Ljava/lang/Object;I)V
 		COMMENT Consumes a resource from the pool of available items.
-		ARG 2 itemId
-			COMMENT the raw id of the item being consumed
-	METHOD m_wkrwvzbp addInput (Ljava/lang/Object;I)V
+		ARG 1 input
+		ARG 2 amount
+	METHOD m_qzcfdjbc hasAny (Ljava/lang/Object;)Z
+		ARG 1 input
+	METHOD m_sbpnsepr account (Ljava/lang/Object;I)V
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/recipe/RecipeMatcher.mapping]: It's not clear at all to me what this does, maybe it's terminology idk about?
Same for methods that call it in `RecipeFinder`.

**Review by supersaiyansubtlety — COMMENTED** [review]
Looks great
Just idk what `account` means as a method name.

File: mappings/net/minecraft/recipe/RecipeMatcher.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/net/minecraft/recipe/RecipeMatcher.mapping]: i'll work on some basic docs for that!

**Review by OroArmor — APPROVED** [review]

File: mappings/net/minecraft/recipe/RecipeMatcher.mapping (hunk 1 above)
Comment (eristhea) [reply mappings/net/minecraft/recipe/RecipeMatcher.mapping]: after considering it i've decided to remove the name in favour of `addInput` -- the original idea was "account for this stack" in matching, but you're right it's not very intuitive

File: simple_type_field_names.json5 [hunk 2]
```diff
@@ -131,6 +133,14 @@
     ]
   },
 
+  // Util
+  "net/minecraft/unmapped/C_fwhrecir": {
```
Comment (supersaiyansubtlety) [inline simple_type_field_names.json5]:
```suggestion
  "net/minecraft/unmapped/C_fwhrecir": { // ContextKey
```

**Review by supersaiyansubtlety — APPROVED** [review]

**Pull request merged**